### ISSUE TITLE
Misc: put .subTitles in a separate component, and make them real HTML headings 

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#4331]
## Feature description and goals

Currently, sub titles are coded as divs with class `subTitle`, and cannot possibly be interpreted as proper titles by assistive technologies. Their font-weight is tweaked to 500, which is not visible on my screen.

To do:
 - create a SubTitle React component
 - use a real HTML heading element
 - maybe set a propert bold level, or another color to make them look like titles
